vm/kvm-self-tests/metadata

```diff
@@ -0,0 +1,12 @@
+[General]
+name=vm/kvm-self-tests
+owner=Luiz Capitulino <lcapitulino@redhat.com>
+description=KVM self tests
+license=GPLv3
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=bc,gcc,make,patch,rsync
```

Comment:
How does this work with dependencies like `python2-lxml`? Don't they need to be specified somehow?

Reply:
Hi @emanchado, please ignore `python2-lxml` and `python3-lxml` as we don't run test case manually now. Those two packages was added by me to Makefile as I once run the test case manually to save time.

Reply:
Ah, cool, thanks. Is this still WIP, or can I have a final look and merge if I don't have any other questions?

Reply:
Please wait for a while as I'm still testing/debugging the changes for `vm/hugepage/libhugetlbfs` after removing its Makefile.